src/components/rounds-list/rounds-list.tsx

```diff
@@ -0,0 +1,39 @@
+import { Round } from "@/shared/types/round"
+import {
+  ParticipantImage,
+  ParticipantItem,
+  ParticipantName,
+  ParticipantsList,
+} from "@/components/participants"
+import { Name } from "@/components/rounds-list"
+
+type RoundsListProps = {
+  rounds: Round[]
+}
+
+export function RoundsList({ rounds }: RoundsListProps) {
```

Comment:
Reading this code again, maybe this component shouldn't exist. Actually, I prefer to make it inline instead of receiving a list of objects, just to make it more flexible or we can split the parts into more components like:
```javascript
<RoundsList>
 {rounds.map((round, index) => (
   <RoundItem>
    <RoundTitle>1º Round</RoundTitle>
    <ParticipantsList>
     // <the same logic for the list> 
    <ParticipantsList>
   </RoundItem>
  ))
</RoundsList>
```
Reading again, makes sense to me that the participants and rounds are related. Maybe we can move the components related participants to the rounds list folder? Something like:
```
sr/components
  rounds-list/
    rounds-list.tsx
    round-item.tsx
    round-title.tsx
    participants-list.tsx
    participant-item.tsx
    participant-name.tsx
    participant-image.tsx
    index.tsx // To export all of the components
```

Reply:
Done! It's better now!